### PR TITLE
feat: Add ARNs part 2

### PR DESCRIPTION
**Pull request by irmatov**

### CODE REVIEW

File: resources/services/apigatewayv2/apis.go [hunk 1]
```diff
@@ -465,7 +465,7 @@ func Apigatewayv2Apis() *schema.Table {
 								Name:        "arn",
 								Description: "The Amazon Resource Name (ARN) for the resource.",
 								Type:        schema.TypeString,
-								Resolver: client.ResolveARN(client.ApigatewayService, func(resource *schema.Resource) ([]string, error) {
+								Resolver: client.ResolveARNWithRegion(client.ApigatewayService, func(resource *schema.Resource) ([]string, error) {
 									r := resource.Item.(types.IntegrationResponse)
 									api := resource.Parent.Parent.Item.(types.Api)
 									return []string{apiIDPart, *api.ApiId, "integrationresponses", *r.IntegrationResponseId}, nil
```
Comment (zagronitay) [inline resources/services/apigatewayv2/apis.go:471]: I think it should be
`
return []string{apiIDPart, *api.ApiId, "integrations", *i.IntegrationID, "integrationresponses", *r.IntegrationResponseId}, nil
`

**Review by zagronitay — CHANGES_REQUESTED** [review]
LGTM
1 minor change

File: resources/services/apigatewayv2/apis.go (hunk 1 above)
Comment (irmatov) [reply resources/services/apigatewayv2/apis.go:471]: Ouch. Fixed.

**Review by zagronitay — APPROVED** [review]
LGTM

**Pull request merged**